app/Http/Controllers/AirlineController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Airline;
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\View\View;
+
+class AirlineController extends Controller
+{
+    public function index(): View
+    {
+        return view('airlines.show', [
+            'links' => Airline::paginate()->links(),
+            'cities' => City::all()
+        ]);
+    }
+
+    public function store(Request $request): JsonResponse
+    {
+        $attributes = $request->validate([
+            'name' => ['required', Rule::unique('airlines')],
+            'description' => ['nullable'],
+            'cities' => ['nullable']
+        ]);
+        $airline = Airline::create([
+            'name' => $attributes['name'],
+            'description' => $attributes['description']
+        ]);
+
+        if (isset($attributes['cities'])) {
+            $cities = City::whereIn('name', $attributes['cities'])->get();
+            $airline->cities()->attach($cities);
+        }
+
+        return response()->json($airline);
+    }
+
+    public function getAirlines(): JsonResponse
+    {
+        $response = [];
+        $airlines = Airline::with('cities')->paginate();
+        $response['data'] = $airlines;
+        return response()->json($response);
```

Comment:
```suggestion
        return response()->json(Airline::with('cities')->paginate());
```
The `paginate` method already has the records within a `data` key, so I don't really see why to add an extra `data` key.